src/org/zaproxy/zap/extension/bruteforce/resources/Messages.properties

```diff
@@ -10,9 +10,12 @@ bruteforce.desc                     = Forced browsing of files and directories u
 bruteforce.dir.popup                = Forced Browse directory
 bruteforce.dir.and.children.popup   = Forced Browse directory (and children)
 bruteforce.options.button.addfile	= Select File...
+bruteforce.options.error.failCaseString.invalid = Invalid Fail Case String.
```

Comment:
We should tell why it's invalid, better say something like "The Fail Case String should not be empty."

Reply:
Is it possible to provide `null` value to the `JTextField` from the UI?
Is it enough to validate only for empty value?

Reply:
No, it just returns an empty string. Yes.